src/components/FileRepo.js

```diff
@@ -145,6 +146,20 @@ const FileRepo = ({ state, effects, ...props }) => {
                 streamData={props.streamData(props.index, props.projectId)}
               />
             </div>
+            {state.modalState.isShown && (
+              <AdvancedFacetViewModal
+                {...{
+                  ...props,
+                  PROJECT_ID: props.projectId,
+                  ES_INDEX: props.index,
+                  API_HOST: process.env.REACT_APP_API,
```

Comment:
netlify seems to be missing this env variable. Is this something we can configure @Jephuff ? :)

Reply:
It was renamed to `REACT_APP_ARRANGER_API`.

You shouldn't need to pass it in though, if AdvancedFacetViewModal uses utils/api to fetch data.

Reply:
@Jephuff :) Ok I refactored I removed the unneeded stuff and refactored Arranger side as part of this pull request: https://github.com/overture-stack/arranger/pull/165